src/Behat/Behat/EventDispatcher/Cli/StopOnFailureController.php

```diff
@@ -32,6 +32,8 @@
 
 /**
  * Stops tests on first scenario failure.
+ * 
+ * TODO this should be moved in the Behat\Testwork\Tester\Cli namespace
```

Comment:
I added this comment as the --stop-on-failure is related to the tester extension, and not the EventDispatcher.

To avoid BC, we might move the class in the Tester namespace and this one could extend the new class and be marked as deprecated (but it might be done in another PR)